client/my-sites/controller.js

```diff
@@ -113,30 +128,39 @@ function renderNoVisibleSites( context ) {
 }
 
 function renderSelectedSiteIsDomainOnly( reactContext, selectedSite ) {
-	const EmptyContentComponent = require( 'components/empty-content' );
+	const FeatureUnavailable = require( 'components/empty-content/feature-unavailable' );
 	const { store: reduxStore } = reactContext;
 
-	removeSidebar( reactContext );
+	renderWithReduxStore( (
+			<FeatureUnavailable selectedSite={ selectedSite } />
+		),
+		document.getElementById( 'primary' ),
+		reduxStore
+	);
 
 	renderWithReduxStore(
-		React.createElement( EmptyContentComponent, {
-			title: i18n.translate( 'Add a site to start using this feature.' ),
-			line: i18n.translate( 'Your domain is only set up with a temporary page. Start a site now to unlock everything WordPress.com can offer.' ),
-			action: i18n.translate( 'Create Site' ),
-			actionURL: `/start/site-selected/?siteSlug=${ encodeURIComponent( selectedSite.slug ) }&siteId=${ encodeURIComponent( selectedSite.ID ) }`,
-			secondaryAction: i18n.translate( 'Manage Domain' ),
-			secondaryActionURL: domainManagementList( selectedSite.slug )
-		} ),
-		document.getElementById( 'primary' ),
+		createNavigation( reactContext ),
+		document.getElementById( 'secondary' ),
 		reduxStore
 	);
 }
 
 function isPathAllowedForDomainOnlySite( pathname, domainName ) {
```

Comment:
`pathname` is provided by the environment (`window.location.pathname`) so I think it might be best not to create a second, properly camelCased name for it.